### DISCUSSION TITLE
Disable Audit Events

### DISCUSSION THREAD

**Discussion by superbarne** [gravitational/teleport#15647] [Q&A]
Hello,

is the a possibility that i disable database query audit events? Or Filter Specific fields?
At the moment the whole query is logged and when i insert data into the database this creates huge files.

Thank you!

**Comment by webvictim** [comment]
I believe functionality is coming to be able to disable audit logging for certain protocols.

Long queries should be truncated inside the audit log, but I presume your issue is that there are many different queries executed which are all logged?

**Reply by superbarne** [reply]
no i have an issue with the audit logs throwing error on the proxy node (helm-kube-agent) and fills up the ram. So i thought if i disable the logging i can get this as a workaround. Now i have compiled myself a teleport version without mongo query logging. Not a good soulution but i wanted to now if it worked and it did.
